### PR TITLE
React-App -Claudia

### PR DESCRIPTION
**Pull request by ckjerrgren**
My reflections on how this week's project turned out:
I would have liked to do something more advanced, but because I spent so much time with the technology this week it had to be something more simple. 

Things I'd like to have clarified or explained in more detail:
EXPO!!
- it really does not want to work for me.

### CODE REVIEW

File: app/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  Text,
+  View,
+  TextInput,
+  ScrollView,
+  TouchableOpacity,
+} from 'react-native';
+
+import Note from './Note';
+
+export default class Main extends React.Component {
```
Comment (EmOlsz) [inline app/components/Main.js:13]: Is there a reason why you used class-based component?

File: app/components/Note.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  Text,
+  View,
+  TouchableOpacity,
+} from 'react-native';
+
+export default class Note extends React.Component {
+  render() {
+    return (
+
+      <View key={this.props.keyval} style={styles.note}>
```
Comment (EmOlsz) [inline app/components/Note.js:13]: No need for `key` attribute here, we do not perform any loop

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: app/components/Main.js (hunk 1 above)
Comment (ckjerrgren) [reply app/components/Main.js:13]: I think endless hours looking at tutorials led me there.

Comment (ckjerrgren) [reply app/components/Main.js:13]: I think endless hours watching tutorials led me there.

**Comment by ckjerrgren** [comment]
the two comments should be fixed now

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**